security/tor/src/opnsense/mvc/app/views/OPNsense/Tor/general.volt

```diff
@@ -30,6 +30,20 @@
 
 <script type="text/javascript">
 
+function tor_update_status() {
+    ajaxCall(url="/api/tor/service/status", sendData={}, callback=function(data,status) {
+        updateServiceStatusUI(data['result']);
```

Comment:
found the bugger, it's 'status'

there is another on this page, but out of the review diff. please fix both.

Reply:
sorry, there used to be two in 0.1. I grepped the wrong repo 👍